ykfr/src/lib.rs

```diff
@@ -306,7 +306,10 @@ impl FrameReconstructor {
                     }
                     SMLocation::Direct(reg, off, _) => {
                         if i == 0 {
-                            // skip first frame
+                            // FIXME: In contrast to indirect locations, we don't need to copy
+                            // values over to the bottom frame. In fact, doing so results in
+                            // segfaults. It is not clear to me as to why that is. Requires
+                            // investigation.
```

Comment:
Curious...

Reply:
I think you should investigate this after this PR is merged while it is still fresh in your mind.

Reply:
I updated this with more informationin [acb97d1](https://github.com/ykjit/yk/pull/637/commits/acb97d1e094316d83e44a89fdea608f5a078a186). There's still some remaining questions, so I left another fixme.